src/App.tsx

```diff
@@ -1,61 +1,44 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
+import { ToDo, ToDoFromServer, UserFromServer } from './types';
+
+const getToDoArray = (): ToDo[] => {
+  return todosFromServer.map((todo: ToDoFromServer)=> {
+    return {
+      ...todo,
+      user: usersFromServer.find((userFromServer: UserFromServer) => {
+        return userFromServer.id === todo.userId;
+      })
+    }
+  }) as ToDo[]
+}
 
 export const App = () => {
+  const [toDoArray , setToDoArray] = useState<ToDo[]>(getToDoArray());
```

Comment:
```suggestion
  const [todos , setTodos] = useState<ToDo[]>(getToDoArray());
```
Just todos is enough